frontend/global/global-styles.scss

```diff
@@ -76,6 +77,9 @@ a.small { // For de-emphasized actions e.g. "Back" button
     margin-right: 1rem;
     text-align: center;
 }
+a, button {
+    cursor: pointer;
```

Comment:
Does the cursor really matter on mobile? :-p

Reply:
Haha neurotic force of habit.